### ISSUE TITLE
Add limited permissions/roles to deprivileged default_service_account

### ISSUE THREAD

**Issue by tbjorkna** [terraform-google-modules/terraform-google-project-factory#773]
### TL;DR

By using `default_service_account = "deprivilege"` I wonder if it would be possible to add new roles and permissions to the default service account after it has been deprivileged? For instance, we may want to strip the Editor role from the default service account, but still let is be used by default by compute engine instances, which may need to use it for Cloud Logging and Cloud Monitoring.

Hence my question, how can I keep using `default_service_account = "deprivilege"` and give the default_service_account the roles [Logs Writer and Monitoring Metric Writer](https://cloud.google.com/logging/docs/agent/logging/troubleshooting#verify_default_service_account_permission)?
